analysis_driver/quality_control/calculate_relatedness.py

```diff
@@ -0,0 +1,68 @@
+from .quality_control_base import QualityControl
+from analysis_driver.config import default as cfg
+from egcg_core import executor
+
+class Relatedness(QualityControl):
+    def __init__(self, dataset, working_dir, gVCF_files, reference, project_id):
+        super().__init__(dataset, working_dir)
+        self.gVCF_files = gVCF_files
+        self.reference = reference
+        self.project_id = project_id
+
+    def get_gatk_genotype_gvcfs_command(self):
+        gVCF_variants = " ". join(["--variant " + i for i in self.gVCF_files])
+        out_prefix = self.project_id + '_genotype_gvcfs'
+        cmd = 'java -jar %s -T GenotypeGVCFs -R %s %s -o %s' % (cfg['tools']['gatk'], self.reference, gVCF_variants, out_prefix)
+        gatk_outfile = out_prefix + '.vcf'
```

Comment:
Can you multithread the GATK call with ```-nt```. set it as a parameter in this function so it is clear when setting the nb cpu used by slurm why you need several.
I would suggest 12 threads and 12 CPU